themes/light/assets/js/config/search.js

```diff
@@ -79,7 +79,9 @@ MEDUSA.config.search = function() {
 
             if (selectedProvider.toLowerCase() === 'utorrent') {
                 client = 'uTorrent';
-                $('#torrent_path_option').hide();
+                $('#label_warning_utorrent').show();
+				        $('#label_anime_warning_utorrent').show();
+				        $('#torrent_path_option').hide();
```

Comment:
These should all be indentented the same amount.

Reply:
Fixed/done.